### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by MICHAELKITH**
### **In this milestone :**
I have added index.html  and style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Product Landing Page - freecodecamp</title>
+</head>
+
+<body>
+  <header id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="image/images.png" alt="logo" class="logo">
+      <ul>
```
Comment (MICHAELKITH) [inline index.html:16]: Good job ! However, I would kindly request you use <li > tag to close every <a> 
--The <li> tag in HTML is used to define the list item in an HTML document..

**Review by MICHAELKITH — COMMENTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @MICHAELKITH , I'm @MICHAELKITH , your reviewer for this PR  👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4



### Changes Required ♻️ 

 Please check all my comments below and make required changes 

Submit for re-review when done 😀 

Thanks 😄 

### Highlights 

1. Descriptive README file ✔️👨🏻‍💻- Good job to set up descriptive readme file

3.No GitHub Flow. ❌ - 

4. Linter errors.  : ❌ - A linter, in simple terms, is a tool that analyzes your code for certain things such as programming errors, bugs, stylistic and formatting errors, and even suspicious entries, You can use this link https://github.com/microverseinc/linters-config/tree/master/html-css

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MICHAELKITH, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Inline comments ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ]  I would advise you to give reasons for your required changes. I can see you commented on the semantic element of the close tag. This is for the student to understand your point of view as the reviewer and not be acting blindly.
- [ ]  I can see you made a list of changes that the student did not do, it is necessary for you to explain so that the student can have an adequate understanding of what you mean and not work in the dark so that the student would know how to go about it. You can help with a link in the case of the linter.
### Where you made the comment
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MICHAELKITH, 👋 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**